Dockerfile

```diff
@@ -17,15 +17,16 @@ RUN apt-get update && apt-get install -y \
 
 # Download Jython
 RUN cd /tmp \
-  && wget -O jython-standalone-$JYTHON_VERSION.jar http://search.maven.org/remotecontent?filepath=org/python/jython-standalone/$JYTHON_VERSION/jython-standalone-$JYTHON_VERSION.jar
+  && wget -O jython-standalone-$JYTHON_VERSION.jar https://search.maven.org/remotecontent?filepath=org/python/jython-standalone/$JYTHON_VERSION/jython-standalone-$JYTHON_VERSION.jar
 
 # Download SaxonPE
 RUN cd /tmp \
-  && wget http://www.saxonica.com/download/SaxonPE$SAXONPE_VERSION.zip \
+  && wget https://www.saxonica.com/download/SaxonPE$SAXONPE_VERSION.zip \
   && unzip SaxonPE$SAXONPE_VERSION.zip
 
 # Copy source files
 COPY fgdc2iso/ /app/
+COPY saxon-license.lic /etc/saxon-license.lic
```

Comment:
We definitely don't want to copy secrets into the container. The container will be published on docker hub.

Reply:
I've updated this mount the license file as a volume.